Sources/Utils/Log.cpp

```diff
@@ -147,8 +148,10 @@ void vLog(int level, char const *classname, char const *funcname,
   fputs(ss.str().c_str(), sOutputStream);
   fflush(sOutputStream);
 
-  if (level == kLogLevelFatal)
+  if (level == kLogLevelFatal) {
+    ds2::Utils::PrintBacktrace();
     abort();
+  }
```

Comment:
And this git blame is why @a20012251 likes curly braces on everything. I'm starting to understand :laughing: